cpt/runner.py

```diff
@@ -125,14 +131,21 @@ def run(self):
                                                         not_export=self.skip_recipe_export,
                                                         update=self._update_dependencies)
                             else:
+                                if self._profile_build_abs_path is not None:
+                                    profile_build = ProfileData([self._profile_build_abs_path],
+                                                                None, None, None)
```

Comment:
```suggestion
                                    profile_build = ProfileData(profiles=[self._profile_build_abs_path], settings=None, options=None, env=None, conf=None)
```

Use parameters when using Conan API, as there is no contract which ensure the method signature.